src/components/Graph.tsx

```diff
@@ -64,7 +64,20 @@ const Graph: React.FC<GraphProps> = ({ data, style = {}, focusedNodeID }) => {
   const divEL = useRef(null);
   const graphData = generateEchartsData(data, focusedNodeID);
   const option = {
-    tooltip: {},
+    tooltip: {
+      trigger: 'item',
+      triggerOn: 'mousemove',
+      formatter: function (params) {
+        if (params.dataType === 'node') {
+          // console.log(params)
+          const nodeDataIndex = params.data.id; // 获取悬停节点的数据索引
+          document.dispatchEvent(new CustomEvent('nodeHovered', { detail: nodeDataIndex }));
+          return nodeDataIndex + '   ' + params.data.value; // 返回自定义的 tooltip 内容
+        } else {
+          return params.data.source + ' > ' + params.data.target + ' : ' + params.data.value;
+        }
+      },
```

Comment:
Thanks for ur suggestion.

I followed your suggestion, but I don't know how to use showTip to determine the movement of the mouse because I tried but didn't succeed. Instead, I used mouseover to implement it. I don't know if the code logic for rendering the table is correct because I have never done a similar task before.